File: dsa-with-java-python/prime.py
class Prime:
    def prime(self):
        for i in range(1,101):
            count=0
            for j in range(1,i+1):
                if i%j==0:
                    count+=1
            if count==2:
                print(i,"is a prime number")
            else:
                print(i,"is not a prime number")


p=Prime()
p.prime()

File: dsa-with-java-python/prime.java
class prime{
    public static void main(String[] args){
        for(int i=2;i<100;i++){
            int count = 0;
            for(int j=1;j<=i;j++){
                if(i%j==0){
                    count++;
                }
            }
            if(count==2){
                System.out.println("prime");
                System.out.println(i);
            }
            else{
                System.out.println("not prime");
                System.out.println(i);
            }
        }
    }
}